content/docs/authorization/configuration/powerflex/_index.md

```diff
@@ -18,23 +18,19 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
 
 2. Edit these parameters in `samples/secret/karavi-authorization-config.json` file in the [CSI PowerFlex](https://github.com/dell/csi-powerflex/tree/main/samples) driver and update/add connection information for one or more backend storage arrays. In an instance where multiple CSI drivers are configured on the same Kubernetes cluster, the port range in the *endpoint* parameter must be different for each driver.
 
-  | Parameter | Description | Required | Default |
+   | Parameter | Description | Required | Default |
    | --------- | ----------- | -------- |-------- |
    | username | Username for connecting to the backend storage array. This parameter is ignored. | No | - |
    | password | Password for connecting to to the backend storage array. This parameter is ignored. | No | - |
    | intendedEndpoint | HTTPS REST API endpoint of the backend storage array. | Yes | - |
-   | endpoint | HTTPS localhost endpoint that the authorization sidecar will listen on. | Yes | https://localhost:9400 |
+   | endpoint | HTTPS localhost endpoint that the authorization sidecar will listen on. This will be the array endpoint to be configured for the driver. | Yes | https://localhost:9400 |
```

Comment:
Seems like what is added newly could be read as an explanation of the first sentence but technically it is saying something different. Can we make it more clear to say it could be either the end point for the auth side car if auth is involved or array end point in other cases.

Reply:
Reverted to the original description for the supported Authorization drivers.